hd/address.go

```diff
@@ -86,8 +85,7 @@ func ComputeTxId(rawTxHex string) string {
 	return HexEncode(ReverseBytes(CalcHash256(HexDecode(rawTxHex))))
 }
 
-// Private methods...
-
+/*
```

Comment:
same thing. why not just delete it if we are not using them?

Reply:
bucky wanted them around

Reply:
ok